db/migrate/20250109165511_remove_taxonomy_and_override_from_filter.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveTaxonomyAndOverrideFromFilter < ActiveRecord::Migration[7.0]
+  def up
+    # remove_column :filters, :override
+    # filters = Filter.where(role_id: Role.where(origin: nil).or(Role.where(builtin: 2)))
+
+    # filters.each do |filter|
+    #   filter.enforce_inherited_taxonomies
+    #   filter.update_column(:taxonomy_search, filter.taxonomy_search)
+    # end
+  end
+
+  def down
+    # add_column :filters, :taxonomy_search, :text
+    # add_column :filters, :override, :boolean, :default => false, :null => false
+    # filters = Filter.where(role_id: Role.where(origin: nil).or(Role.where(builtin: 2))).where(override: false).where(taxonomy_search: nil)
+
+    # filters.each do |filter|
+    #   filter.build_taxonomy_search
```

Comment:
Maybe it's better to use the same logic from `up`?
```suggestion
    #   filter.enforce_inherited_taxonomies
```